modules/service/src/main/scala/ch/epfl/bluebrain/nexus/iam/service/Main.scala

```diff
@@ -114,6 +116,15 @@ object Main {
           logger.info(s"Bound to '${binding.localAddress.getHostString}': '${binding.localAddress.getPort}'")
         case _ => Await.result(as.terminate(), 3.seconds)
       }
+
+      KafkaPublisher.start(
+        "permission",
```

Comment:
Can we have all these literals defined as constants somewhere?

Reply:
Probably some of it, e.g. could/should be moved to config